src/main/java/com/griddynamics/jagger/jenkins/jaas/plugin/JaggerTestExecutionBuilder.java

```diff
@@ -223,6 +225,38 @@ private TestExecutionEntity waitTestExecutionFinished(PrintStream logger, Long e
         return execution;
     }
 
+    private void publishReportLink(PrintStream logger, TestExecutionEntity executionFinished) {
+        if (executionFinished.getSessionId() == null)
+            logger.println("sessionId is unavailable. Can’t publish link to the test report.");
+        else
+            logger.println("Test execution report can be found by the link " + evaluatedJaasEndpoint + "/report?sessionId=" + executionFinished.getSessionId());
+    }
+
+    private void checkDecision(PrintStream logger, String sessionId, RestTemplate restTemplate) throws AbortException {
+        try {
+            logger.println();
+            logger.println(format("Checking decision for test session with id=%s ... ", sessionId));
+            RequestEntity<?> requestEntity = RequestEntity.get(new URI(evaluatedJaasEndpoint + "/sessions/" + sessionId + "/decision")).build();
+            ResponseEntity<DecisionPerSessionDto> responseEntity = restTemplate.exchange(requestEntity, DecisionPerSessionDto.class);
+            Decision decision = responseEntity.getBody().getDecision();
+            logger.println(format("Decision for session with id=%s is %s", sessionId, decision));
+            if (decision != Decision.OK) {
+                String decisionJson = new ObjectMapper().writeValueAsString(responseEntity.getBody());
+                logger.println("Full session decision:\n" + decisionJson);
```

Comment:
let's log full session decision for OK sessions as well. First details, then "Decision for session with id "